releasenotes/notes/remove-deprecated-factorizers-linear-solvers-4631870129749624.yaml

```diff
@@ -0,0 +1,9 @@
+---
+upgrade:
+  - |
+    The deprecated modules ``factorizers`` and ``linear_solvers`` have been removed from
+    :mod:`qiskit.algorithms`. These arguments
+    were originally deprecated as part of the 0.22.0 release (released on
+    October 13, 2022). You can access the code throguht the textbook tutorials instead:
+    `Linear Solvers (HHL) <https://qiskit.org/textbook/ch-applications/hhl_tutorial.html>`_ ,
+    `Factorizers (Shor) <https://qiskit.org/textbook/ch-algorithms/shor.html>`_
```

Comment:
Well that accessing the code is true of HHL, but not Shor. The Shor code that was in Qiskit is removed - the code in the textbook implementing Shor is unrelated.

Better urls now are (better in the sense this is the new textbook and only this one has the link to the HHL code plus the older textbook is still installing HHL from Terra and has not been updated).

https://learn.qiskit.org/course/ch-applications/solving-linear-systems-of-equations-using-hhl-and-its-qiskit-implementation  (Where the code formerly in Qiskit Terra can be accessed via the link at the start of this section `A. Running HHL on a simulator: general method`

https://learn.qiskit.org/course/ch-algorithms/shors-algorithm